stts/Services/StatusPage/Packet.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Packet.swift
+//  stts
+//
+//  Created by D on 1/6/17.
+//  Copyright © 2017 inket. All rights reserved.
+//
```

Comment:
Can you remove the extra copyright comments (lines 5-7) so that it matches the other files? Thanks.